CHANGES.md

```diff
@@ -97,14 +97,14 @@ OpenSSL 3.0
      such an argument, such as `EVP_CIPHER_fetch` and similar algorithm
      fetching functions.
    - Indirectly, by creating a new library context and then assigning
-     it as the new default, with `OPENSSL_CTX_set0_default`.
+     it as the new default, with `OSSL_LIB_CTX_set0_default`.
 
-   All public OpenSSL functions that take an `OPENSSL_CTX` pointer,
-   apart from the functions directly related to `OPENSSL_CTX`, accept
+   All public OpenSSL functions that take an `OSSL_LIB_CTX` pointer,
+   apart from the functions directly related to `OSSL_LIB_CTX`, accept
    NULL to indicate that the default library context should be used.
 
    Library code that changes the default library context using
-   `OPENSSL_CTX_set0_default` should take care to restore it with a
+   `OSSL_LIB_CTX_set0_default` should take care to restore it with a
    second call before returning to the caller.
```

Comment:
TODO: all renamings in CHANGES.md and NEWS.md need to be reverted. Instead, add a CHANGES entry about the renaming.

Reply:
I changed my mind: instead of reverting the renaming, I added a note to CHANGES.md (but not to NEWS.md), see the [force-push](https://github.com/openssl/openssl/compare/56dac49bf761b14c9538367939d14887b393ef3f..b62480a8589cb6d6b18a1e871ad9dd1587eb1450) diff.